packages/0x.js/test/token_wrapper_test.ts

```diff
@@ -515,3 +519,14 @@ describe('TokenWrapper', () => {
     });
 });
 // tslint:disable:max-file-line-count
+
+function addEmptyWalletSubprovider(provider: Web3.Provider): Web3.Provider {
```

Comment:
Let's make a name more generic. I'd love to use that function in other places.

Reply:
I think it doesn't make sense for now, but I'll generaize it later to prepend any subprovider.